src/ansible_navigator/image_manager/puller.py

```diff
@@ -14,7 +14,8 @@
 from ..utils.functions import shlex_join
 
 
-class ImageAssessment(SimpleNamespace):
+@dataclass
```

Comment:
Freeze? Or does it get mutated?

Reply:
It gets incrementally updated as the process goes:

e.g.:
`self._assessment.pull_required = pull`

Reply:
Now noted in the description with frozen = False